src/lib/moment/momentUtils.ts

```diff
@@ -5,3 +5,14 @@ import moment from 'moment';
 export const formatUTCDateStringToLocal = (dateStringUTC: string): string => {
   return moment(dateStringUTC).local().format('YYYY-MM-DD HH:mm:ss');
 };
+
+//takes in a string of a date time YYYY-MM-DD format. Returns true if its before the current month.
+export const lessThanCurrentMonth = (dateString: string): boolean => {
```

Comment:
Function doesn't consider january --> december. We need to consider periods rather than months (months aren't enough info). Tiffany merged some helpful moment functions if you want to pull (you might not even need to create a function if you're able to do the logic properly with Tiffany's existing util functions).

Reply:
oh I didnt thake this into consideration ty